inc/admin/namespace.php

```diff
@@ -102,28 +102,23 @@ function dispatch() {
  * Render the list page.
  */
 function render() {
-	global $wp_list_table;
-
-	?>
+	global $wp_list_table; ?>
 	<div class="wrap">
 		<h2>
 			<?php
 			esc_html_e( 'Registered Applications', 'oauth2' );
 
-			if ( current_user_can( 'create_users' ) ) : ?>
+			if ( current_user_can( 'create_users' ) ) { ?>
 				<a href="<?php echo esc_url( get_url( 'action=add' ) ) ?>"
 				   class="add-new-h2"><?php echo esc_html_x( 'Add New', 'application', 'oauth2' ); ?></a>
-				<?php
-			endif;
-			?>
+			<?php } ?>
```

Comment:
These should use the long if/endif format; per [the coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style) it's a bit more readable when interspersed with HTML.